specification.markdown

```diff
@@ -579,7 +579,7 @@ The maxver attribute is OPTIONAL. If the maxver attribute is not present then th
 
 ### \<documentation> Element
 
-Within an EBML Schema, the XPath of `<documentation>` attribute is `/EBMLSchema/element/documentation`.
+Within an EBML Schema, the XPath of `<documentation>` element is `/EBMLSchema/element/documentation`.
```

Comment:
Shouldn't this be plural or contain an article? "…the XPath of `<documentation>` elements…" or "…the XPath of the `<documentation>` element…" At the moment it simply sounds grammatically wrong, especially given that the very next sentence does use an article.

Same for the other instances in this commit.

Reply:
Given there's only one path it seems correct to use singular here.